nipype/interfaces/ants/tests/test_auto_Registration.py

```diff
@@ -1,157 +0,0 @@
-# AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
```

Comment:
I think we still wanna keep this

Reply:
I didn't remove it manually. I mentioned in #2349 that since we have a non-auto test, this test has been removed by `make specs`. What happen when you run `make specs`?